src/CMakeLists.txt

```
@@ -1,7 +1,15 @@
 cmake_minimum_required(VERSION 3.2)
 project(emulator)
 
-set(SOURCE_FILES main.cpp)
+set(TARGET_NAME emulator)
 
-add_executable(emulator ${SOURCE_FILES})
+set(SOURCE_FILES main.cpp
+        emulator/Emulator.cpp
+        emulator/Emulator.hpp
+        cartridge/Cartridge.cpp
+        cartridge/Cartridge.hpp)
+
+add_executable(emulator ${SOURCE_FILES} ${COMMON})
 install(TARGETS emulator DESTINATION ${EMULATOR_INSTALL_BIN_DIR})
+
+target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_SOURCE_DIR}/src/cartridge)
```

On peut faire 
```suggestion
target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_SOURCE_DIR}/src)
```
Et mettre tous nos include avec le directory (`cartidge/my.hpp`), comme ça on voit où est le fichier
T'en penses quoi ?